bch/config/mainnet-bch.conf

```diff
@@ -0,0 +1,14 @@
+txindex=1
+dnsseed=0
```

Comment:
Please remove that since it would not allow the node to use DNS seeds for the initial list of peers